source/_integrations/smartenergy_goecharger.markdown

```diff
@@ -0,0 +1,61 @@
+| `charging_allowed`           | Whether the car is allowed to charge at all.                     |
+| `charger_max_current`        | Requested current for charging in A.                             |
+| `energy_since_car_connected` | Energy in kWh since car is connected.                            |
+| `energy_total`               | Total energy used in kWh.                                        |
+| `phase_switch_mode`          | Phase switch mode - auto/1/3.                                    |
+| `phases_number_connected`    | Number of connected phases - relates to the `phase_switch_mode`. |
+| `charger_access`             | Access control for the device - 0/1.                             |
+| `name`                       | Friendly name of the device.                                     |
+
+## Services
+
+Following functions are exposed as Home Assistant services, thus can be used by other integrations or in automations.
+
+| Name                    | Parameters                                                                                         | Description                                                                                                                                                    |
+| ----------------------- | -------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `start_charging`        | `{"device_name": "example_charger", "charging_power": 10}` or `{"device_name": "example_charger"}` | Starts charging with a specified charging power. `charging_power` is optional and you can use this service purely to start without setting the charging power. |
+| `stop_charging`         | `{"device_name": "example_charger"}`                                                               | Stop charging.                                                                                                                                                 |
+| `change_charging_power` | `{"device_name": "example_charger", "charging_power": 10}`                                         | Change charging power for a given charger.                                                                                                                     |
+| `set_phase`             | `{"device_name": "example_charger", "phase": 1}`                                                   | Change phase for a given charger. `phase` accepts values 0, 1, 2.                                                                                              |
+| `set_transaction`       | `{"device_name": "example_charger", "status": 0}`                                                  | Set the wallbox transaction. `status` accepts values None (no transaction) and 0 (authenticate all users).                                                     |
```

Comment:
This indicates the parent PR is larger than is has to be.
Services should be added in follow-up PRs.